app/services/valid_test_data_generators/applications_populater.rb

```diff
@@ -14,6 +14,7 @@ def populate
       logger.info "ApplicationsPopulater: Started!"
 
       ActiveRecord::Base.transaction do
+        create_schedules!
```

Comment:
can we leave those from this service, as I think we will only create them once outside this?